BootloaderCorePkg/Library/AcpiInitLib/AcpiInitLib.c

```diff
@@ -423,27 +420,13 @@ UpdateFwst (
   // Fill in details for FWST ACPI table
   //
   FwstAcpiTablePtr = (EFI_FWST_ACPI_DESCRIPTION_TABLE *)Current;
-
-  CopyMem(&FwstAcpiTablePtr->EsrtTableEntry.FwClass, &gEsrtSystemFirmwareGuid, sizeof (EFI_GUID)); 
-  FwstAcpiTablePtr->EsrtTableEntry.FwType = ESRT_FW_TYPE_SYSTEMFIRMWARE;
-
-  //
-  // Get current version and lowest supported SVN version
-  //
-  if (LdrGlobal->VerInfoPtr != NULL) {
-    Version = LdrGlobal->VerInfoPtr;
-    FwstAcpiTablePtr->EsrtTableEntry.FwVersion = Version->ImageVersion.SecureVerNum;
```

Comment:
where FwstAcpiTablePtr->EsrtTableEntry.FwVersion is set?

Reply:
This is a to do part for me. Will update in the next version

Reply:
it is ok for me. please create an issue to track it.

Reply:
Done Created one for this